packages/oid4vci-holder/README.md

```diff
@@ -12,7 +12,53 @@
 
 ---
 
-TODO
+## Usage
+
+### OPENID_INITIATE_ISSUANCE SCHEMA
+
+pre-authorized_code
+
+```typescript
+agent.oid4vciHolderGetInitiationData({
+  requestData: {
```

Comment:
Please don't use ancient protocols in the readme

Reply:
When I read "ancient", I could imagine people reading the specs of the protocol from a parchment, discovered in some archaeological site :smile:. Removed.